docs/cvl/v5-changes.md

```diff
@@ -193,6 +194,52 @@ is the main contract being verified, then passing `--contract C` will cause
 method variables to be instantiated in the same way the would have in older
 versions.
 
+Disallow Solidity function calls in quantified expressions
+----------------------------------------------------------
+
+Starting with `certora-cli` version 5.0, the Prover no longer supports
+making Solidity contract method calls in quantified expression bodies by
+default. For example, given the simple contract below, you can no longer
+use the `method` `foo()` in a quantified expression body.
+
+```{code-block} solidity
```

Comment:
I don't feel strongly about this, but I would consider condensing this; something like "if `foo` is a contract method, then a quantified expression like `require (forall uint i . i == foo(i))` will no longer compile.

Reply:
Although on second though, you may want to keep the contract code so that you can refer to it below; I'll add a comment there.